lib/hades/mentorships/mentorships.ex

```diff
@@ -139,4 +137,57 @@ defmodule Hades.Mentorships do
     |> Mentoree.changeset(attrs)
     |> Repo.update()
   end
+
+  @doc """
+  Creates a request.
+
+  ## Examples
+
+      iex> create_request(%{field: value})
+      {:ok, %Request{}}
+
+      iex> create_request(%{field: bad_value})
+      {:error, %Ecto.Changeset{}}
+
+  """
+  def create_request(attrs \\ %{}) do
+    %Request{}
+    |> Request.changeset(attrs)
+    |> Repo.insert
+  end
+
+  def request_status(%Request{} = request) do
+    status = request.status
+    if status == Request.status[:active] do :active else :inactive end
+  end
+
+  def request_skill_area(%Request{} = request) do
+    skill_area = request.skill_area
+    cond do
```

Comment:
This is really not Open/Closed =/

Reply:
@webofbits perhaps we can think of a better way of making this comparison?
Is it possible to check if the value is contained within a list ?

I think ideally these values should be hardcoded only in one place.
Is it perhaps possible to reuse the list in https://github.com/lbighetti/hades/blob/master/lib/hades/mentorships/mentor.ex ?

` @allowed_skill_areas ~w(Backend Frontend DevOps UX/UI Mobile Fullstack)`

Reply:
I am not quite sure if the enum approach is a best choice here but since the status, mentoree_level should have only one value we can't use a list I think. 
Maybe we can use a boolean field for status like `request.is_opened` (true/false).
The `mentoree_level` needs to be an attribute with multiple choices like an enum in rails, but since we don't have the same feature on elixir, I think we should try to use something common on elixir idk.
I am in favor to close this and organize how it should be done before start coding like I did.